book/07-git-tools/sections/debugging.asc

```diff
@@ -1,16 +1,16 @@
-=== Debugging with Git
+=== Pagdebug gamit ang Git
 
-In addition to being primarily for version control, Git also provides a couple commands to help you debug your source code projects.
-Because Git is designed to handle nearly any type of content, these tools are pretty generic, but they can often help you hunt for a bug or culprit when things go wrong.
+Sa dagdag sa pagiging pangunahin para sa kontrol ng bersyon, Ang Git ay nagbibigay rin ng ilang utos para tumulong sa pag-debug sa iyong source code na mga proyekto.
+Dahil ang Git ay dinisenyo upang pangasiwaan ang halos anumang uri ng nilalaman, itong mga kasangkapan ay medyo generic, ngunit maaari nilang madalas na makatulong sayo upang maghanap ng isang bug o salarin kapang ang mga bagay ay nagkakamali.
 
 [[_file_annotation]]
-==== File Annotation
+==== Ang Annotation ng File
 
-If you track down a bug in your code and want to know when it was introduced and why, file annotation is often your best tool.
-It shows you what commit was the last to modify each line of any file.
-So if you see that a method in your code is buggy, you can annotate the file with `git blame` to determine which commit was responsible for the introduction of that line.
+Kung ikaw ay sumusubaybay ng isang bug sa iyong code at gusto mong malaman kapag ito ay ipinakilala at bakit, ang annotation ng file ay madalas na iyong pinakamahusay na kasankapan.
+Ito ay nagpapakita sa yo kung anong commit ang huling binago sa bawat linya ng anumang file.
+Kaya kung ikaw ay nakakita ng paraan sa iyong code ay buggy, maaari mong i-annotate ang file gamit ang `git blame` para malaman kung anong commit ang responsable para sa pagpapakilala sa linyang iyon.
 
-The following example uses `git blame` to determine which commit and committer was responsible for lines in the top-level Linux kernel `Makefile` and, further, uses the `-L` option to restrict the output of the annotation to lines 69 through 82 of that file:
+Para sa sumusunod na halimbawa ay gumagamit ng `git blame` para matukoy kung anong commit at nag-commit ang responsable para sa mga linya sa itaas-antas na Linux kernel na `Makefile` at, higit pa, gumagamit ng `-L` na opsyon upang paghigpitan ang output sa annotation ng mga linya 69 na sa pamamagitan sa 82 sa file na iyon:
```

Comment:
`top-level` could be `nangungunang antas` or `itaas na antas`

Reply:
> linya 69 na sa pamamagitan sa 82 sa file na iyon

could be

> linya 69 hanggang 82 sa file na iyon

Reply:
change using itaas na antas
done change